### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JYOSHREDDY**
1) Created a link pointing to Pygeoweaver encoraging people with python background to check that repo.
2) Created a link to check out the new website - https://geoweaver.dev/ — features of geoweaver.

### CODE REVIEW

**Comment by JYOSHREDDY** [comment]
3) Added a Geoweaver banner.

File: README.md [hunk 1]
```diff
@@ -1,69 +1,289 @@
...
+  - Intercept websocket traffic to save notebook versions, enabling easy revision history access.
+
+4) **Process History and Logging**:
+  - Detailed history of every process run, including logs and outputs, is stored.
+
+5) **Workflow Management**:
+  - Link processes to create workflows for parallel or sequential execution across different resources.
+  - All aspects of workflow management are centralized within GeoWeaver.
+
+# Geoweaver Installation Guide
```
Comment (ZihengSun) [inline README.md:67]: please keep this installation guide as usual. Don't add these detailed steps in the front page. When people are interested, they should click on the install guide link which was here.

**Review by ZihengSun — CHANGES_REQUESTED** [review]
Don't add detailed steps. Restored the old links to the separated markdown docs.

**Review by ZihengSun — APPROVED** [review]
LG

**Pull request merged**